components/org.wso2.identity.apps.taglibs.layout.controller/src/main/java/org/wso2/identity/apps/taglibs/layout/controller/core/TemplateEngine.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.apps.taglibs.layout.controller.core;
+
+import java.io.Serializable;
+import java.io.Writer;
+import java.net.URL;
+import java.util.Map;
+
+/**
+ * Interface for the template engine
```

Comment:
```suggestion
 * Interface for the template engine.
```
Missing full stop. Check other places as well